### PR TITLE
Codecov report upload step is added

### PR DESCRIPTION
**Pull request by alisengul53**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -51,3 +51,8 @@ jobs:
                   HAZELCAST_ENTERPRISE_KEY: ${{ secrets.HAZELCAST_ENTERPRISE_KEY }}
               run: |
                   npm run coverage
+            - name: Publish to Codecov
+              if: ${{matrix.os == 'ubuntu-latest'}}
```
Comment (srknzl) [inline .github/workflows/ci.yml:55]: nit: for consistency we can add spaces 
```suggestion
              if: ${{ matrix.os == 'ubuntu-latest' }}
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -51,3 +51,8 @@ jobs:
...
+              uses: codecov/codecov-action@v2
+              with:
+                files: coverage/cobertura-coverage.xml
```
Comment (srknzl) [inline .github/workflows/ci.yml:58]: nit: should we add a newline to the end of file?

**Review by srknzl — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (alisengul53) [reply .github/workflows/ci.yml]: New line added

**Comment by srknzl** [comment]
verify

**Pull request merged**